deploy/ci-infra.Jenkinsfile

```diff
@@ -0,0 +1,63 @@
+def legionVersion = null
+
+node {
+    try {
+        stage('Terminate Cluster if exists') {
```

Comment:
@alexander-semenets remove it since cluster will be terminated in finally section